PRASCore/src/Systems/SystemModel.jl

```diff
@@ -71,7 +76,7 @@ function SystemModel(
           "time zone for the system timestamps, provide a range of " *
           "`ZonedDateTime` instead of `DateTime`."
 
-    utc = TimeZone("UTC")
+    utc = tz"UTC"
     time_start = ZonedDateTime(first(timestamps), utc)
```

Comment:
The time zone requirement is to avoid data ambiguity in large multi-timezone systems, and I think it's worth preserving even if it adds a bit more friction in smaller systems. I'm open to revisiting this in the rewrite, but for the sake of continuity let's not change it here?